content/02.introduction.md

```diff
@@ -24,6 +24,74 @@ We designed a series of experiments to determine the extent to which label funct
 We examine relationships that indicate similar types of physical interactions (i.e., gene-binds-gene and compound-binds-gene) as well as different types (i.e., disease-associates-gene and compound-treats-disease).
 The re-use of label functions could dramatically reduce the number required to generate and update a heterogeneous knowledge graph.
 
-## Recent Work
+### Related Work
 
-Talk about what has been done in the field in regards to text mining and knowledge base integration
+Relationship extraction is the process of detecting and classifying semantic relationships from a collection of text.
+This process can be broken down into three different categories: (1) the use of natural language processing techniques such as manually crafted rules and identifying key text patterns for relationship extraction, (2) the use of unsupervised methods via co-occurrence scores or clustering, and (3) supervised or semi-supervised machine learning using annotated datasets for classification of documents or sentences.
+In this section, we discuss selected efforts for each type of edge that we include in this project.
+
+#### Disease-Gene Associations 
+
+Efforts to extract Disease-associates-Gene (DaG) relationships have often used manually crafted rules or unsupervised methods.
+One study used hand crafted rules based on a sentence's grammatical structure, represented as dependency trees, to extract DaG relationships [@tag:pkde4j].
+Some of these rules inspired certain DaG text pattern label functions in our work.
+Another study used co-occurrence frequencies within abstracts and sentences to score the likelihood of association between disease and gene pairs [@tag:diseases].
+The results of this study were incorporated into Hetionet v1, so this served as one of our distant supervision label functions.
+Another approach built off of the above work by incorporating a supervised classifier, trained via distant supervision, into their scoring scheme [@tag:cocoscore].
+Each sentence containing a disease and gene mention is scored using a logistic regression model.
```

Comment:
```suggestion
Each sentence containing a disease and gene mention is scored using a logistic regression model and combined using the same co-occurrence approach used in [@tag:diseases].
```
This should be `first author's last name et al. [@tag:diseases].` Otherwise it'll just show up as a number in the document.